packages/test/stochastic-test-utils/package.json

```diff
@@ -62,7 +62,7 @@
     "best-random": "^1.0.0"
   },
   "devDependencies": {
-    "@fluid-tools/benchmark": "^0.40.0",
+    "@fluid-tools/benchmark": "^0.42.0",
```

Comment:
Can't help but notice the bump went from 0.40 -> 0.42

Reply:
Just wanted to skip the-number-who-must-not-be-named 😝 . Yeah, not sure why we didn't update that dependency everywhere when we released 0.41 back in May. Just bringing them up to latest now.